src/VisualStudio/Core/Def/Implementation/DebuggerIntelliSense/DebuggerIntellisenseWorkspace.cs

```diff
@@ -7,20 +7,20 @@
 
 namespace Microsoft.VisualStudio.LanguageServices.Implementation.DebuggerIntelliSense
 {
-    internal class DebuggerIntelliSenseWorkspace : Microsoft.CodeAnalysis.Workspace
+    internal sealed class DebuggerIntelliSenseWorkspace : Workspace
     {
         public DebuggerIntelliSenseWorkspace(Solution solution)
-            : base(solution.Workspace.Services.HostServices, "DebbugerIntellisense")
```

Comment:
I am of course terrified of what might depend on this constant that has a spelling error. C'est la vie.